runtime/port/unix/j9sysinfo.c

```diff
@@ -860,15 +860,13 @@ getS390Description(struct J9PortLibrary *portLibrary, J9ProcessorDesc *desc)
 #endif /* defined(S390) && defined(LINUX) */
 
 	/* HIGH_GPRS support */
-#if defined(OMR_ENV_DATA64)
```

Comment:
Pointing this out explicitly to @joransiu and @NigelYiboYu who has recently got bitten by this.

Reply:
I'm going to need to think some more about this one. I'm not sure we actually want HPRs to be enabled on 31-bit platforms because there are other dependencies in the VM for example where we do not support collectable references being in HPRs. I'm not sure we want to go down this road to be honest. I'll have to think some more.

Reply:
Marking this WIP until tomorrow. Going to sleep on it.

Reply:
So I've done extensive thinking and testing on this and I've decided it is a good idea to enable HPR support on 31-bit Linux as well. This has been completed in 4522048